sdap/processors/extracttimestampprocessor.py

```diff
@@ -0,0 +1,63 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import nexusproto
+from nexusproto.serialization import from_shaped_array
+
+import datetime
+import time
+import logging
+from netCDF4 import Dataset, num2date
+from pytz import timezone
+
+from sdap.processors import NexusTileProcessor
+
+EPOCH = timezone('UTC').localize(datetime.datetime(1970, 1, 1))
+
+
+def to_seconds_from_epoch(timestamp, pattern='%Y-%m-%dT%H:%M:%S'):
+    try:
+        seconds = int(time.mktime(time.strptime(timestamp[:19], pattern)))
```

Comment:
For the other datasets, the timestamp is already provided as an integer, which the datetime library can parse and be offset by a start date.  But this one is in the form `YYYY-mm-ddTHH:MM:SS` and needs to be converted first.

Reply:
I can add a check for seconds as integer vs timestamp as string format first.  Then if given an integer and seconds from start date, I can use the original functions to parse date, otherwise it needs to be converted, which defaults to seconds since epoch.

Reply:
Yea I know what you mean. I was more thinking the use of `time.mktime` vs. `(datetime.datetime - EPOCH).total_seconds()` like we do elsewhere. Second thought, sticking with `time.mktime` is fine; more succinct anyways.